notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1118_ims_slewing_azimuth_analysis_positioning.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
```

Comment:
<p>Same here. Could you add labels to the red lines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>